ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml

```diff
@@ -50,11 +50,23 @@ tests:
     workflow: baremetalds-e2e-ovn-ipv6
 - as: e2e-metal-ipi-ovn-dualstack
   optional: true
+  skip_if_only_changed: (^(docs|.github)/)|(\.md$)|((^|/)OWNERS$)
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e-ovn-dualstack
+- as: e2e-metal-ipi-serial-ipv4
+  skip_if_only_changed: (^(docs|.github)/)|(\.md$)|((^|/)OWNERS$)
```

Comment:
let's also add DCO and LICENSE here and in all the other skip_if_only_changed entries

Reply:
Good idea, thanks